easybuild/easyconfigs/s/smallgenomeutilities/smallgenomeutilities-0.2.1-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,49 @@
+# EasyBuild EasyConfig 
+#
+# John Dey jfdey@fredhutch.org fizwit@github.com
+easyblock = 'PythonPackage'
+
+name = 'smallgenomeutilities'
+version = '0.2.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/cbg-ethz/smallgenomeutilities'
+description = """The smallgenomeutilities are a collection of scripts that is
+ useful for dealing and manipulating NGS data of small viral genomes. They
+ are written in Python 3 with a small number of dependencies."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://github.com/cbg-ethz/smallgenomeutilities/archive']
+sources = [{
+    'download_filename': 'master.zip',
+    'filename': SOURCE_ZIP
+}]
```

Comment:
@fizwit Instead of downloading `master.zip`, which could easily be replaced if a newer release is created, download `0.2.1.tar.gz`

Reply:
Yes, I will add the version.  I was the one who created the issue to request that they create git tags, and they did.